src/pages/Main/Main.js

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Main.scss';
+import PrdList from '../components/productlist';
+import Filter from '../components/Filter';
 
 function Main() {
-  return <p>Hello worl1d</p>;
+  const [data, setData] = useState([]);
+  return (
+    <div className="main">
```

Comment:
```suggestion
  const [productData, setProductData] = useState([]);
  
  return (
    <div className="main">
```
state의 이름도 해당 state가 어떤 상태값을 가지고 있는지 이름으로 명확하게 작성해주셔야 합니다!

추가로 가독성을 위해 한칸 띄워주세요~